packer/ansible/templates/pmm2.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=PMM2 Docker container
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+ExecStartPre=-/usr/bin/docker exec pmm-server stop
+ExecStartPre=-/usr/bin/docker rm pmm-server
```

Comment:
Why do we remove pmm-server on each start? Looks like if the user upgrades PMM using the UI button and then restart the service, the service might use an incorrect image.